server/db/models/enhancement.js

```diff
@@ -0,0 +1,41 @@
+const crypto = require('crypto');
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Animal = db.define('animal', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  description: {
+    type: Sequelize.STRING,
+    allowNull: false,
+
+  },
+  imageUrl: {
+    type: Sequelize.STRING,
+  },
+  tags: {
```

Comment:
I think in wikistack we had the user enter the tags as a string and then used a hook to store it as an array.

Not sure what we want to do. Could add an issue to fix it later.